core/domain/wipeout_service.py

```diff
@@ -1253,3 +1254,91 @@ def _pseudonymize_models_transactional(voiceover_application_models):
             voiceover_application_models[
                 i:i + feconf.MAX_NUMBER_OF_OPS_IN_TRANSACTION]
         )
+
+
+def _pseudonymize_blog_posts_models(pending_deletion_request):
+    """Pseudonymize the blog post models for the user with user_id.
+
+    Args:
+        pending_deletion_request:
+            PendingDeletionRequest. The pending deletion
+            request object to be saved in the datastore.
```

Comment:
Please keep the type together with the argument, here and elsewhere, like so:
```suggestion
        pending_deletion_request: PendingDeletionRequest. The pending deletion
            request object to be saved in the datastore.
```

Reply:
Done.